lib/ClangImporter/Serializability.cpp

```diff
@@ -315,6 +315,10 @@ namespace {
       if (loc.isValid())
         IsSerializable = false;
     }
+
+    void writeBTFTypeTagAttr(const clang::BTFTypeTagAttr *attr) {
+      IsSerializable = false;
```

Comment:
After talking to @rjmccall about this, I'll put up another PR after this one to instead handle `BTFTagAttributeType` in `DataStreamBasicWriter::writeQualType` by grabbing the underlying type and merging its qualifiers, then convert this to a fatal error since it should never occur. This is good enough to at least get us compiling though :)

Reply:
Went ahead and did this now since I was waiting on clang fixes anyway.